src/encoded/region_search.py

```diff
@@ -17,27 +19,19 @@
 
 _ENSEMBL_URL = 'http://rest.ensembl.org/'
 
-_REGION_FIELDS = [
-    'embedded.files.uuid',
-    'embedded.files.accession',
-    'embedded.files.href',
-    'embedded.files.file_format',
-    'embedded.files.assembly',
-    'embedded.files.output_type',
-    'embedded.files.derived_from'
-]
-
 _FACETS = [
     ('assay_term_name', {'title': 'Assay'}),
     ('biosample_ontology.term_name', {'title': 'Biosample term'}),
     ('target.label', {'title': 'Target'}),
-    ('replicates.library.biosample.donor.organism.scientific_name', {
-        'title': 'Organism'
-    }),
     ('biosample_ontology.organ_slims', {'title': 'Organ'}),
     ('biosample_ontology.cell_slims', {'title': 'Cell'}),
-    ('assembly', {'title': 'Genome assembly'}),
-    ('files.file_type', {'title': 'Available data'})
+    ('assembly', {'title': 'Genome assembly'}) # necessary for aggregating data
+]
+
+_REMOVE_FACETS = [
```

Comment:
seems odd that these have to be explicitly deleted.

Reply:
yes, if we remove them before search, a few aggregations won't work, we have to filter them out after getting the results